tests/test_transforms/test_outliers/test_outliers_transform.py

```diff
@@ -421,9 +444,18 @@ def test_full_pipeline(transform, outliers_solid_tsds):
         (MedianOutliersTransform(in_column="target", ignore_flag_column="is_holiday")),
         (DensityOutliersTransform(in_column="target", ignore_flag_column="is_holiday")),
         (PredictionIntervalOutliersTransform(in_column="target", model="sarimax", ignore_flag_column="is_holiday")),
+        (IForestOutlierTransform(in_column="target", ignore_flag_column="is_holiday")),
     ],
 )
 def test_advance_usage_data_in_transform_nonregressor(transform, outliers_solid_tsds_non_regressor_holiday):
     ts = outliers_solid_tsds_non_regressor_holiday
     transform.fit(ts)
     _ = transform.transform(ts)
+
+
+def test_isolation_forest_with_features_from_transforms(outliers_solid_tsds):
```

Comment:
Lets add several test cases with `DateFlagsTransform`. This transform is essential to `IForestOutlierTransform`.